### PR TITLE
Prep Javascript Functions First Draft Nelson Daza

### PR DESCRIPTION
**Pull request by neldaza**
Screen Shot 2021-09-22 at 5 44 55 PM

### CODE REVIEW

File: prep-javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+console.log(resultSubtractNumbers);
+
+function getCircleCircumference(radius) {
+  return 2 * Math.PI * radius;
+}
+var circumference = getCircleCircumference(5);
+console.log(circumference);
+
+function getFullName(firstName, lastName) {
+  return firstName + lastName;
```
Comment (scbowler) [inline prep-javascript-functions/main.js]: Here you want a space between the first and last name

File: prep-javascript-functions/main.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+var name = getFullName('Nelson ', 'Daza');
```
Comment (scbowler) [inline prep-javascript-functions/main.js]: Adding a space at the end of the first name isn't ideal. People using your function wouldn't know to do that or remember to do that. The space should be added by the function as mentioned above

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but 1 update needed. Let me know if you have any questions.

**Comment by neldaza** [comment]
updated!

Screen Shot 2021-09-23 at 6 35 00 PM

File: prep-javascript-functions/main.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+function convertHoursToMinutes(hours) {
+  return hours * 60;
+}
+var minutes = convertHoursToMinutes(5);
+console.log('minutes', minutes);
+
+function getGreeting(name) {
+  return 'Hello ' + name + '!';
+}
+var Greeting = getGreeting('Nelson');
```
Comment (scbowler) [inline prep-javascript-functions/main.js:16]: We want to always start variable and function names with lowercase letter.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍 

1 FYI comment, no changes required

**Pull request merged**